### DISCUSSION TITLE
Introducción a El Modelado de Procesos de Negocio BPMN

### DISCUSSION THREAD

**Discussion by carlos-gmx3** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#115] [Show and tell]
Un nuevo aporte
En la escuela ando viendo la notación de de modelado de procesos de negocio y aquí les vengo a dejar una breve introducción y 2 guías de modelado muuuuuy completas
![imagen](https://user-images.githubusercontent.com/99064239/155625363-68ba4f0f-936f-4153-8892-fbdf4f86c8dd.png)
![imagen](https://user-images.githubusercontent.com/99064239/155625425-f46039fd-ed65-489c-83cf-03f1b44fd279.png)

1. [BPMN: Guía de Referencia y Modelado](https://users.dcc.uchile.cl/~nbaloian/DSS-DCC/Software/ModeladoBPMN.pdf)

2. [GUÍA PARA MODELAR PROCESOS Y
PROCEDIMIENTOS BAJO EL ESTÁNDAR
BPMN 2.0](https://info.jalisco.gob.mx/sites/default/files/leyes/dom-p286-d2_001_guia_para_modelar_procesos_y_procedimientos_001.pdf)

Si tienen dudas, adelante, les ayudaré en lo que pueda

**Comment by AlamMancera** [comment]
Que joya hermano, muchas gracias!

**Comment by OscarPedroza** [comment]
Muchas gracias por el aporte compañero
